README.md

```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
+tap "homebrew/core"
```

Comment:
This is a no-op effectively as it'll never be output by `brew bundle dump` and Homebrew taps it implicitly. I say remove it.